CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # Changelog
+
+- [#11] Added filename to error messages
```

Comment:
11? but this is PR#19.  did you also want to reference an issue?

Reply:
good catch, yea the issue is 11 which should get auto closed since it was referenced in commit. but this should probably be 19